tests/unittest/CFGProgrammer/CFGProgrammer_test.cpp

```diff
@@ -821,10 +821,34 @@ TEST(ProgrammerHelper, PrintDeviceListSimpleTest) {
   printDeviceList(cable, deviceList);
   std::string output = testing::internal::GetCapturedStdout();
   std::string expected =
-  "INFO: Cable               | Device\n"
-  "INFO: -----------------------------------------------\n"
-  "INFO: (1) RsFtdi_1_2        (1) Device1\n"
-  "INFO: (1) RsFtdi_1_2        (2) Gemini2\n";
+  "INFO: Cable               | Device            | Flash Size\n"
```

Comment:
Is there a better way to verify this than to hard-code the string which is dummy anyways ?

Reply:
This unit test is supposed to decouple the hardware from testing. 
These tests string value is a dummy, but this string is the expected value if it is run with real hardware. The unit test is simulating the real data here.

Reply:
We shall have integration tests when hardware is available..